### PR TITLE
Add pages aa1 to implement-1

### PR DESCRIPTION
**Pull request by Aichnerc**
**Checklist:**

 
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [ ] Have you updated or added documentation for the change?
 - [x] Have you tested your changes with successful results?

**Type of Changes:**

 
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [x] New feature (non-breaking change which adds functionality)
 - [ ] Documentation (non-breaking change which adds documentation)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)

**What is the current behavior? (link to any open issues here)**
- Currently there is no page selection in the SideNav.
- Currently there is no page section.

**What is the new behavior (if this is a feature change)?**
- The pages now appear in the SideNav.
- A basic page section has been added which queries the page by handle.

### CODE REVIEW

File: src/components/molecules/SideNav/index.jsx [hunk 1]
```diff
@@ -64,6 +72,9 @@ class SideNavbar extends React.Component {
     // Tell the parent component if expanded or not
     this.props.handleToggle(expNormalized);
 
+    // Get names of pages
```
Comment (schettn) [inline src/components/molecules/SideNav/index.jsx]:
```suggestion
    this.props.handleToggle(expNormalized);
    // Get names of pages
```

**Review by schettn — APPROVED** [review]
Looks good

**Review by kleberbaum — APPROVED** [review]
Should be ok to merge.

**Review by pinterid — APPROVED** [review]
Looks good

**Pull request merged**